mod app;
mod client;

use app::error::Error;

#[tokio::main]
async fn main() -> Result<(), Error> {
    
    Ok(())
}